clients/java/src/main/java/io/camunda/client/ClientProperties.java

```diff
@@ -28,135 +28,129 @@ public final class ClientProperties {
    * @see CamundaClientBuilder#applyEnvironmentVariableOverrides(boolean)
    */
   public static final String APPLY_ENVIRONMENT_VARIABLES_OVERRIDES =
-      "zeebe.client.applyEnvironmentVariableOverrides";
-
-  /**
-   * @deprecated since 8.5 for removal with 8.8, replaced by {@link ClientProperties#GRPC_ADDRESS}
-   * @see CamundaClientBuilder#gatewayAddress(String)
-   */
-  @Deprecated public static final String GATEWAY_ADDRESS = "zeebe.client.gateway.address";
+      "client.applyEnvironmentVariableOverrides";
 
   /**
    * @deprecated since 8.5 for removal with 8.8, where toggling between both will not be possible
    * @see CamundaClientBuilder#preferRestOverGrpc(boolean)
    */
   @Deprecated
-  public static final String PREFER_REST_OVER_GRPC = "zeebe.client.gateway.preferRestOverGrpc";
+  public static final String PREFER_REST_OVER_GRPC = "client.gateway.preferRestOverGrpc";
 
   /**
    * @see CamundaClientBuilder#restAddress(URI)
    */
-  public static final String REST_ADDRESS = "zeebe.client.gateway.rest.address";
+  public static final String REST_ADDRESS = "client.gateway.rest.address";
 
   /**
    * @see CamundaClientBuilder#grpcAddress(URI)
    */
-  public static final String GRPC_ADDRESS = "zeebe.client.gateway.grpc.address";
+  public static final String GRPC_ADDRESS = "client.gateway.grpc.address";
 
   /**
    * @see CamundaClientBuilder#defaultTenantId(String)
    */
-  public static final String DEFAULT_TENANT_ID = "zeebe.client.tenantId";
+  public static final String DEFAULT_TENANT_ID = "client.tenantId";
 
   /**
    * @see CamundaClientBuilder#defaultJobWorkerTenantIds(List)
    */
-  public static final String DEFAULT_JOB_WORKER_TENANT_IDS = "zeebe.client.worker.tenantIds";
+  public static final String DEFAULT_JOB_WORKER_TENANT_IDS = "client.zeebe.worker.tenantIds";
```

Comment:
Why is `zeebe` added back here? As the client is integrated with the camunda api, I see no reason in sticking to the term "zeebe" at all.